src/assets/scripts/client/aircraft/Pilot/Pilot.js

```diff
@@ -114,7 +114,7 @@ export default class Pilot {
      * @param airportModel {AirportModel}
      * @return {array}            [success of operation, readback]
      */
-    maintainAltitude(currentAltitude, altitude, expedite, shouldUseSoftCeiling, airportModel) {
+    maintainAltitude(currentAltitude, altitude, expedite, shouldUseSoftCeiling, airportModel, aircraftModel) {
```

Comment:
Could you add a `TODO` here to look at this later. 6 parameters is too many, there may be an easier way. This can be handled later though.